app/src/main/java/com/jerboa/ui/components/community/CommunityActivity.kt

```diff
@@ -235,6 +236,11 @@ fun CommunityActivity(
                                                             auth = acct.jwt,
                                                         ),
                                                     )
+                                                    siteViewModel.getSite(
```

Comment:
Does this guarantee that getSite only is executed once followCommunity is done?

Reply:
Fixed!